calico-cloud/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -64,16 +64,17 @@ The default tier is where:
 
 ## System tiers
 
-System tiers are added during installation and are hidden by default.
+System tiers are added during installation and are hidden by default. Create all of your tiers after `allow-tigera` and `tigera-security`.
```

Comment:
I'm not sure what this means. The new sentence makes it sounds like you can create allow-tigera and tigera-security, and that you must do this first. But the preceding sentence says this happens at installation. 

Can you create tiers before allow-tigera is created?

Reply:
The guidance for users is to create their tiers after the system tiers (in terms of left to right tier ordering). Tweaked text.